finger-exercises-react/react/tic-tac-toe/src/redux/index.js

```diff
@@ -1,11 +1,15 @@
 import thunk from 'redux-thunk';
+import { reducer as form } from 'redux-form';
 
 import { combineReducers } from '../../node_modules/redux';
 
 import game from './reducer';
 import prevGames from './PrevGames/reducer';
+import logged from './Login/reducer';
```

Comment:
Use the same name as the reducer here `login`, it's easier to find related code 😄